### ISSUE TITLE
Line 50 in gm_vae.py

### ISSUE THREAD

**Issue by wjgan7** [RuiShu/vae-clustering#11]
Newbie here. Maybe this is a basic question, but why is it that line 50 in gm_vae.py does
```python
nent = -cross_entropy_with_logits(qy_logit, qy)
```
Shouldn't we pass in the label, y, as the second parameter?
